packages/datadog-instrumentations/src/kafkajs.js

```diff
@@ -16,10 +16,19 @@ const consumerFinishCh = channel('apm:kafkajs:consume:finish')
 const consumerErrorCh = channel('apm:kafkajs:consume:error')
 
 addHook({ name: 'kafkajs', versions: ['>=1.4'] }, (obj) => {
-  const Kafka = obj.Kafka
+  class Kafka extends obj.Kafka {
+    constructor (options) {
+      super(options)
+      this.__brokers = (options.brokers && typeof options.brokers !== 'function')
```

Comment:
Usually we use two underscores for weird meta stuff. Since this is just a "hidden" property let's go with a single underscore.